includes/definitions/eltex-mes23xx.yaml

```diff
@@ -22,3 +27,8 @@ bad_iftype:
     - tunnel
     - softwareLoopback
     - propPointToPointSerial
+
+oids:
+    no_bulk:
+        - rlPethPsePortOutputPower
```

Comment:
Can you please change these to use the fully qualified OID name?
`MARVELL-POE-MIB::rlPethPsePortOutputPower`
`MARVELL-POE-MIB::rlPethPsePortPowerLimit`

(and use that in the snmpwalk too)

Reply:
> Can you please change these to use the fully qualified OID name? `MARVELL-POE-MIB::rlPethPsePortOutputPower` `MARVELL-POE-MIB::rlPethPsePortPowerLimit`
> 
> (and use that in the snmpwalk too)

function gen_snmpwalk_cmd($device, $oids, $options = null, $mib = null, $mibdir = null)
{
    d_echo(Config::getCombined($device['os'], 'oids.no_bulk'));

output:    
 rlPethPsePortOutputPower 
array (
  0 => 'MARVELL-POE-MIB::rlPethPsePortOutputPower',
  1 => 'MARVELL-POE-MIB::rlPethPsePortPowerLimit',
)  

so array is there

but looks like it is matched only aganist $oids
|| ! empty(array_intersect(Arr::wrap($oids), Config::getCombined($device['os'], 'oids.no_bulk'))))) 

maybe i am wrong ...

Reply:
> Can you please change these to use the fully qualified OID name? `MARVELL-POE-MIB::rlPethPsePortOutputPower` `MARVELL-POE-MIB::rlPethPsePortPowerLimit`
> 
> (and use that in the snmpwalk too)

ok, yes, you are right
if i put MARVELL-POE-MIB:: in walk, then MARVELL-POE-MIB:: will work in YAML
sorry for noise